src/cmd/config.go

```diff
@@ -70,16 +71,19 @@ func init() {
 }
 
 func LoadConfig() (*common.Config, error) {
-	v := &common.ConfigVersion{}
-	viper.Unmarshal(&v)
-	if v.Version != common.ConfigCurrentVersion {
-		return nil, fmt.Errorf("supported config version is '%s' but found '%s' | Please update config file or create a new sample with `kubectl opslevel config sample`", common.ConfigCurrentVersion, v.Version)
+	var commonConfig common.Config
+
+	yamlData, err := os.ReadFile(cfgFile)
```

Comment:
Need to port over the logic from root.go `readConfig()` to:

1. If `cfgFile == "-"` read the config from stdin as yaml
2. If the file at location `cfgFile` is found, read it. This is how the code works now.
3. If the file at location `cfgFile` is not found, read the file `$HOME/opslevel.yaml`